api/report_server.thrift

```diff
@@ -21,58 +21,55 @@ const string API_VERSION = '6.0'
 const i64 MAX_QUERY_SIZE = 500
 //=================================================
 
-//-----------------------------------------------------------------------------
-struct RunData{
+struct RunData {
   1: i64            runId,        // unique id of the run
```

Comment:
Maybe we could adopt the convention of having full sentences in comments to comply with the rest of the codebase as the part of this revamp?